Digital_Signature/README.md

```diff
@@ -0,0 +1,4 @@
+# digsig-windows
+Middleware written in Go that allows a browser to interact with the windows cert store via CNG
+
```

Comment:
Please update with an explicit "Current Status" to signify that this is still in a prototype stage.